understanding/22/redundant-entry.html

```diff
@@ -13,6 +13,16 @@ <h2>Status</h2>
       <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
   </section>
 
+  <section id="brief">
+   <h2>In brief</h2>
+   <dl>
+      <dt>Goal</dt><dd>Make it easier for users to complete multi-step processes.</dd>
+      <dt>What to do</dt><dd>Don't ask for the same information twice in the same session.</dd>
+      <dt>Why it's important</dt><dd>Some people with cognitive disabilities have difficulty remember what they entered before.</dd>
```

Comment:
```suggestion
      <dt>Why it's important</dt><dd>Some people with cognitive disabilities have difficulty remembering what they entered before, or may make a mistake when re-entering the same information more than once.</dd>
```

Reply:
How about 
```suggestion
      <dt>Why it's important</dt><dd>Some people with cognitive disabilities have difficulty remember or re-entering what they entered before.</dd>
```

Reply:
fair enough, i mis-remembered (hah) that the understanding touched on the fact that if you ask the same thing twice, you're doubling the chances of somebody getting it accidentally wrong.

however, the change from "remember" to "remembering" is still needed though